app/helpers/hyrax/doi/work_form_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Hyrax
+  module DOI
+    module WorkFormHelper
+      def form_tabs_for(form:)
+        if Flipflop.doi_tab? && form.model_class.ancestors.include?(Hyrax::DOI::DOIBehavior)
```

Comment:
This is the only time you're not negating the method; would it be better to rename the feature to be `inline_doi` and not have to negative it through out the PR?

Reply:
Ah, yes. That's a good idea.  I could not think of an opposite name for "DOI tab" that wouldn't make it more confusing. But your suggestion will do it!

Reply:
Addressed below